### PR TITLE
Update the game section

### PR DESCRIPTION
**Pull request by Extloga**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -253,14 +253,14 @@ A curated list of Nintendo 64 development resources including toolchains, docume
 
 * [AeroGauge](https://github.com/LLONSIT/AeroGauge) - An in-progress decompilation of _AeroGauge_
 * [Aidyn Chronicles](https://github.com/blackgamma7/Aidyn) - An in-progress decompilation of _Aidyn Chronicles_
-* [Banjo-Kazooie](https://gitlab.com/banjo.decomp/banjo-kazooie) - An in-progress decompilation of _Banjo-Kazooie_
+* [Banjo-Kazooie](https://gitlab.com/banjo.decomp/banjo-kazooie) - An **complete** decompilation of _Banjo-Kazooie_
```
Comment (bigbass1997) [inline README.md:256]: Minor typo. Change `An` to `A`.

Comment (networkfusion) [reply README.md:256]: Complete is not really a true statement.

Comment (bigbass1997) [reply README.md:256]: What is the status of the project if not complete?

Comment (networkfusion) [reply README.md]: > What is the status of the project if not complete?

The decomp has matched the "NTSC V1.0" ROM from ASM to C. PAL + all other ROM versions and docs are not yet done.

Comment (command-tab) [reply README.md]: Should I (or Extloga) change it back to "in-progress", then?

Comment (networkfusion) [reply README.md]: > Should I (or Extloga) change it back to "in-progress", then?

IMO, it is kinda the same situation as OOT, so, yes.

Comment (Extloga) [reply README.md]: I thought that one version is enough to define a project as complete in this repository. I followed the example of previous commits, where Perfect Dark was defined as complete in 2022 and this definition seems to be even more lenient as not even one version reached 100 percent at the time: Only the essential functions were matched but not the bytes and the project stalled below 100 percent. My threshold for declaring completion is that one version reached 100 percent.

I think that it is a good idea to use this opportunity to clarify the definition once and for all:

If we use the definition that all versions matched, I will change the status of Banjo-Kazooie, Paper Mario, and Perfect Dark.
If we use the definition that one version matched, I will change the status of Perfect Dark.
If we use the definition that the functions of one version matched, I will not change anything.

The owner of the repository should decide this question.

Comment (command-tab) [reply README.md]: I don’t mind “complete” allowing for some wiggle room. One can make an argument about the extent to which the term applies, but for the sake of this repo, I think it’s enough for “complete” to mean effectively all the functions for at least one version of the title — enough to use as a fully working and understandable architectural reference for new development purposes. I’m certainly open to arguments in other directions, though 🙂

Comment (Extloga) [reply README.md]: > I don’t mind “complete” allowing for some wiggle room. One can make an argument about the extent to which the term applies, but for the sake of this repo, I think it’s enough for “complete” to mean effectively all the functions for at least one version of the title — enough to use as a fully working and understandable architectural reference for new development purposes. I’m certainly open to arguments in other directions, though 🙂

That is a reasonable argument. I changed the list to be in accordance with this definition. Further discussions about this question could take place in a separate issue, but preliminarily, I would say that this PR is ready to merge.

My position is clear, but I am not dogmatic in this question and will go along with whatever is ultimately decided.

Comment (command-tab) [reply README.md]: Awesome! Thank you!

**Pull request merged**